model-optimizer/extensions/middle/AddMeanScaleValues_test.py

```diff
@@ -27,9 +27,11 @@
 
 nodes = {
     **regular_op_with_shaped_data('parameter', [1, 3, 227, 227],
-                                  {'type': 'Parameter', 'op': 'Parameter', 'shape': [1, 3, 227, 227]}),
+                                  {'type': 'Parameter', 'op': 'Parameter', 'shape': [1, 3, 227, 227],
+                                   'fw_tensor_debug_info': 'Placeholder'}),
```

Comment:
Is the `fw_tensor_debug_info` attribute set for the data node? I think that it is set for the operation node, which is not what we want.